.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```

Comment:
```suggestion

```

that gets me as well every time. Had it solved temporarily by doing https://github.com/bisq-network/bisq/pull/3205, yet, it just reappeared again.

Reply:
oh whoops... sorry. I thought I had fixed this but it came back :-(